utils/debug.c

```diff
@@ -136,7 +153,16 @@ void setup_color(enum color_setting color, char *pager)
 		out_color = color;
 	}
 
-	if (out_color != COLOR_ON) {
+	if (format_mode == FORMAT_HTML) {
```

Comment:
Ditto.

Reply:
As mentioned, we can just dump the plain text if someone wants to print without color so I think this is okay.